packages/core/botpress-builtins/src/renderers/builtin_action-button.js

```diff
@@ -15,13 +15,25 @@ export default data => {
               return {
                 type: 'postback',
                 title: a.title,
-                payload: a.title
+                payload: a.text
               }
             } else if (a.action === 'Open URL') {
```

Comment:
maybe ```switch(){}``` will be look better

Reply:
Again, if we want to make radical changes to the code structure styles, this probably should be a separate PR... I just added to existing structure...